backend/tests/core/DatabaseHelper.cs

```diff
@@ -50,6 +51,7 @@ public static PimsContext CreatePimsContext(this TestHelper helper, string dbNam
             helper.AddSingleton(user);
             var options = new DbContextOptionsBuilder<PimsContext>()
                 .UseInMemoryDatabase(databaseName: dbName)
+                .ConfigureWarnings(m => m.Ignore(InMemoryEventId.TransactionIgnoredWarning))
```

Comment:
For some reason EF started throwing warnings during unit-testing.  This ignores those.